auxx/src/Command/Rollback.hs

```diff
@@ -55,7 +58,7 @@ rollbackAndDump numToRollback outFile = do
             liftIO $ BSL.writeFile outFile (serialize allTxs)
             logInfo $ sformat ("Dumped "%int%" transactions to "%string)
                       (length allTxs) (outFile)
-            rollbackBlocksUnsafe (BypassSecurityCheck True) blunds
+            rollbackBlocksUnsafe (BypassSecurityCheck True) (ShouldCallBListener True) blunds
```

Comment:
EDIT: Philipp's comment was preserved (sorry, I thought it got lost). Ignore this comment then 😉 

A minor one but I still think @kantp 's idea was nice enough to be ported-over here: Why not coalesce that `Bool` into a sum type directly, like the following?

```
data BListenerAction = Call
                                     | DontCall
```

(I suck at naming things, but you get the idea). Essentially, wrapping a `Bool` in an another type constructor will give you only marginal gains in terms of type safety (although _will_ cure boolean blindness). At this point, why not scrap the wrapped `Bool` entirely?

Nothing that will stop approving the PR at this stage, but I'm curious to hear what people think!

Reply:
I like `data ShouldCallBListener = CallBListener | DontCallBListener`, but please don't do it in this PR, because in master we already have `ShouldCallBListener Bool`.

Reply:
Fair enough, in which case I would be happy enough with a new issue being created on YT as commitment we are going to do that 😀